Issue 300 tests from 1 to 5

arthurmkrtchian opened the pull request:
Write tests for:

1. CategoryDtoMapper
2. CategoryDtoResponseMapper
3. CustomHabitMapper
4. CustomShoppingListMapper
5. CustomShoppingListResponseDtoMapper

marhino8 committed:
Issue 300 tests from 1 to 5 for:
1. CategoryDtoMapper
2. CategoryDtoResponseMapper
3. CustomHabitMapper
4. CustomShoppingListMapper
5. CustomShoppingListResponseDtoMapper

arthurmkrtchian committed:
Issue 300 tests from 1 to 5 for:
1. CategoryDtoMapper
2. CategoryDtoResponseMapper
3. CustomHabitMapper
4. CustomShoppingListMapper
5. CustomShoppingListResponseDtoMapper

arthurmkrtchian committed:
Issue 300 tests from 1 to 5 for:
1. CategoryDtoMapper
2. CategoryDtoResponseMapper
3. CustomHabitMapper
4. CustomShoppingListMapper
5. CustomShoppingListResponseDtoMapper

MaksymFartushok reviewed (approved):
All tests pass successfully, coverage 100%. 
Great!)

arthurmkrtchian committed:
Issue 300 tests from 1 to 5 for:
1. CategoryDtoMapper
2. CategoryDtoResponseMapper
3. CustomHabitMapper
4. CustomShoppingListMapper
5. CustomShoppingListResponseDtoMapper

otulush reviewed (approved):
Excellent. All tests passed, 100% coverage.
